vscode/src/webviewPanel.ts

```diff
@@ -315,7 +315,7 @@ export function registerWebViewCommands(context: ExtensionContext) {
         await worker.run(code, "", parseInt(numberOfShots), evtTarget);
         clearTimeout(compilerTimeout);
       } catch (e: any) {
-        log.error("Codegen error. ", e.toString());
+        log.error("Histogram error. ", e.toString());
```

Comment:
Was this just something you noticed but not exactly related to the fix?

Reply:
Yes. The dialog box that pops up on histogram execution failures had the wrong strong, and it made me thing I had run the wrong command!